swap/protocol.go

```diff
@@ -98,6 +101,10 @@ func (s *Swap) verifyHandshake(msg interface{}) error {
 		return ErrEmptyAddressInSignature
 	}
 
+	if handshake.ChainID != s.chainID {
```

Comment:
Could you update the comment to the function `verifyHandshake`? Right now it says that it is only verifying the chequebook address transmitted

Reply:
done